conf/messages.en

```diff
@@ -168,14 +169,13 @@ login.error.password.length = Password must be 100 characters or less
 login.error.auth = Needs to log in
 
 welcome.title = Welcome
-welcome.heading = Welcome,
+welcome.heading = Choose a task
 welcome.BA = Billing Authority Reports
 welcome.want.to = I want to
-welcome.councilTax.url = Council Tax
-welcome.councilTax.description = Submit your Council Tax reports and view your previous Council Tax submissions.
-submissionCategory.council_tax_webform = create a Council Tax report
-submissionCategory.council_tax_xml_upload = Upload a file of Council Tax reports
-
+welcome.council_tax_webform = Create a report
+welcome.council_tax_webform.description = Text to describe the action.
+welcome.council_tax_upload = Upload a report
+welcome.council_tax_upload.description = Text to describe the action.
```

Comment:
"Text to describe the action." seems like very vague content. Has this been passed by a content designer? I'm making this comment without knowing the layout and context of the page of course.

Reply:
The content hasn't been confirmed by Rhys! It'll be confirmed at a letter date...
It's based on this prototype:
https://voa-bar-prototype.herokuapp.com/ - (user: voa / p/w: hmrc) - and billing authorities BA1445

Reply:
ok